src/layers/core/point-density-grid-layer/index.js

```diff
@@ -0,0 +1 @@
+export {default as default} from './point-density-grid-layer';
```

Comment:
FWIW, if you look at the top level `index.js` files, we've started exporting layers directly from xyz-layer/xyz-layer.js, that way we could drop all these one line index files.